test/test_individual.py

```diff
@@ -158,25 +185,26 @@ def test_to_and_from_torch_plus_backprop():
         3,
         ID_NON_CODING_GENE,
     ]
-    individual = IndividualSingleGenome(None, genome)
+    individual = _create_individual(None, genome, ind_type)
 
     def f_target(x):
         return math.pi * x
 
     f = individual.to_torch()
```

Comment:
shouldn't this just use the `_unpack_evaluation`?

Reply:
This is a bit tricky but the problem is that we need the packed `f` for the `update_parameters_from_torch_class` call below.

Reply:
ah, i see. and creating two instances is not possible, as you need the optimizer and the `update_...` function to work on the same one, right? ok, let's leave it like this.